### PR TITLE
add old server to ServerSwitchEvent

### PR DESCRIPTION
**Pull request by Chronophylos**
We need this information and it is logical to me to have the old server in a ServerSwitchEvent

### CODE REVIEW

File: api/src/main/java/net/md_5/bungee/api/event/ServerSwitchEvent.java [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import lombok.ToString;
 import net.md_5.bungee.api.connection.ProxiedPlayer;
 import net.md_5.bungee.api.plugin.Event;
+import net.md_5.bungee.api.connection.Server;
```
Comment (Black-Hole) [inline api/src/main/java/net/md_5/bungee/api/event/ServerSwitchEvent.java]: BungeeCord requires imports to be sorted.

Comment (Chronophylos) [reply api/src/main/java/net/md_5/bungee/api/event/ServerSwitchEvent.java]: I fixed that, thank you

**Comment by Mystiflow** [comment]
Also linked to https://github.com/SpigotMC/BungeeCord/pull/2379 but I think I prefer this because it returns Server over ServerInfo for more control

**Comment by md-5** [comment]
The old server is disconnected at this point though, its useless over ServerInfo

**Comment by Chronophylos** [comment]
I'd really appreciate it if you'd merge either one of the PRs

**Comment by md-5** [comment]
#2379 merged, thanks anyway

**Pull request closed without merging**